doc/permissions.md

```diff
@@ -0,0 +1,173 @@
+# Access controls
+
+## Introduction
+Astronomical data is often subject to complex data rights policies. As a data platform designed to ingest and serve data from multiple experiments and groups, each potentially with different access policies, SkyPortal must be able to enforce arbitrary logic governing who can see, interact with, and modify data. SkyPortal enforces such policies using a custom row-level security (RLS) framework in the API layer.
+
+RLS allows SkyPortal developers to define policies that restrict, with row- and user / token-level granularity, which rows of a table (e.g., photometry, spectra, groups, followup requests, data streams, etc.) that a user can read, update, create, and delete. SkyPortal uses baselayer's ORM-based framework for RLS within API transactions. With baselayer's RLS framework, SkyPortal developers can
+
+* Be sure that access policies will be automatically and consistently enforced whenever a protected record is accessed in an API transaction
```

Comment:
only if the custom methods are explicitly used in the handler

Reply:
yep, removed "automatically" here.